### PR TITLE
Removes all wrapped errors in dynamodb.go

### PR DESCRIPTION
**Pull request by epps**
# Description

Removes all wrapped errors in dynamodb.go.

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have fixed any merge conflicts

### CODE REVIEW

File: provider/dynamodb.go [hunk 1]
```diff
@@ -113,17 +114,17 @@ func CreateMetadataTable(dynamodbClient *dynamodb.DynamoDB) error {
 	}
 	_, err := dynamodbClient.DescribeTable(describeMetadataTableParams)
 	if err != nil {
-		fmt.Println("Could not describe dynamo metadata table, attemping to create...", err)
+		fmt.Println("Could not describe dynamo metadata table, attempting to create...", err)
```
Comment (sdreyer) [inline provider/dynamodb.go:117]: Do we have the log library in here?

File: provider/dynamodb.go [hunk 2]
```diff
@@ -113,17 +114,17 @@ func CreateMetadataTable(dynamodbClient *dynamodb.DynamoDB) error {
...
 	} else {
 		return nil
 	}
 	_, err = dynamodbClient.CreateTable(params)
 	if err != nil {
-		return fmt.Errorf("create attempt: %v", err)
+		return fferr.NewExecutionError(pt.DynamoDBOnline.String(), "", "", "", err)
```
Comment (sdreyer) [inline provider/dynamodb.go:123]: need a better format for general execution errors

**Review by sdreyer — COMMENTED** [review]
Need to figure out that generalize execution error

File: provider/dynamodb.go (hunk 1 above)
Comment (epps) [reply provider/dynamodb.go:117]: We do not, and actually many of the other provider do not either (e.g. Redis). I'm happy to add logging here, though.

Comment (epps) [reply provider/dynamodb.go:117]: Logging added to DynamoDB.

File: provider/dynamodb.go [hunk 3]
```diff
@@ -70,13 +73,14 @@ func NewDynamodbOnlineStore(options *pc.DynamodbConfig) (*dynamodbOnlineStore, e
 	}
 	sess := session.Must(session.NewSession(config))
 	dynamodbClient := dynamodb.New(sess)
-	if err := CreateMetadataTable(dynamodbClient); err != nil {
+	logger := logging.NewLogger("provider")
```
Comment (sdreyer) [inline provider/dynamodb.go:76]: Should change this to dynamodb probably

**Review by sdreyer — APPROVED** [review]

**Pull request merged**